test/help.test.js

```diff
@@ -45,6 +46,22 @@ describe('loopback generator help', function() {
     process.env.SLC_COMMAND = undefined;
   });
 
+  it('print right help message for each generator', function() {
+    process.env.SLC_COMMAND = 'loopback --help';
+    var names = ['acl', 'app', 'boot-script', 'datasource', 'export-api-def',
```

Comment:
Perhaps `command`, `cmd` or `cmdName` would be a more descriptive variable name?